### DISCUSSION TITLE
heavy augmentation 코드

### DISCUSSION THREAD

**Discussion by jiwoo0212** [boostcampaitech2/object-detection-level2-cv-06#127] [ideas]
mmdetection 에서 albumentation 을 이용할 수 있어 이를 이용해 대표적으로 많이 사용되는 augmentation을 해봤는데요!

PR을 하기에는 아직 실험중이어서 파일이 여러버전이라 

사용하실분들은 heavy_augmentation branch 코드를 참고해주세요!

[코드](https://github.com/boostcampaitech2/object-detection-level2-cv-06/blob/heavy_augmentation/mmconfig/datasets/team_base_dataset_aug.py)

참고로 type=pascal voc 되어있는 거 오타아닙니다..! coco dataset도 저렇게 넣어줘야하더라구여

**Comment by jaehwan-AI** [comment]
albumentation의 one of 디폴트 확률이 제가 알기로 0.5인데 0.2나 0.3으로 조정해서 사용하는 이유가 있을까요?
heavy augmentation이면 augmentation들이 더 높은 확률로 적용이 되어야 하는 것이 아닌가 라는 생각으로 질문드립니다.

**Reply by jiwoo0212** [reply]
@jaehwan-AI  이론적인 근거가 있진 않고, 아래 첫번째 링크가 albumentation 예제이고,  두번째 글도 참고를 했었는데 다들 확률을 크게 주지않아서 이를 따라갔습니다. 

또한 augmenatation 종류가 많아서 확률을 크게 줘도될까..? 라는 생각이 들었던 것 같아요

확률을 크게 줘서 실험해봐도 좋을 것 같습니다!!

 https://github.com/open-mmlab/mmdetection/blob/master/configs/albu_example/mask_rcnn_r50_fpn_albu_1x_coco.py

https://deep-learning-study.tistory.com/673

**Reply by jaehwan-AI** [reply]
따로 참고하신 자료가 있었군요.
질문을 한 이유는 augmentation은 사용자가 정의하기 나름이라 따로 의도한 바가 있는지가 궁금했던 것이었습니다.
답변 감사합니다
